### DISCUSSION TITLE
Window position

### DISCUSSION THREAD

**Discussion by Davefdm** [yermak/AudioBookConverter#390] [Q&A]
When using version 6.0.1 Portable the title bar and bottom are off screen. There is no way to move the window. I can use WinInfo Portable to temporarily move it into view but after exiting then starting again, the title bar is out of view again.
This is on Windows 10 64bit.
I was only able to get one conversion out of it. Now when I run the program the main window shows but the Add Files dialog doesn't come up. The only thing that works is Exit.
Version 2 would be better if it had all the formats.
I decided to go with freac portable.
https://portableapps.com/apps/music_video/freac_portable

**Comment by yermak** [comment]
What screen size (inches), resolution and scale settings do you have?
